cmd/wrapcontract/usage.md

```diff
@@ -2,9 +2,26 @@ This command takes the runtime bytecode, the bytecode deployed on-chain, as inpu
 
 ```bash
 $ polycli wrap-contract 69602a60005260206000f3600052600a6016f3
+$ echo 69602a60005260206000f3600052600a6016f3 | polycli wrap-contract 
 
 ```
 
+You can also provide a path to a file, and the bytecode while be read from there.
+
+```bash
+$ polycli wrap-contract bytecode.txt
+$ polycli wrap-contract ../bytecode.txt
+$ polycli wrap-contract /tmp/bytecode.txt
+$ echo /tmp/bytecode.txt | polycli wrap-contract
```

Comment:
```suggestion
$ cat /tmp/bytecode.txt | polycli wrap-contract
```

Reply:
A cat here would not show what I want to show :-)
The requested feature was to be able to read the content from the file, so for that I need the echo, so the command reads the path to the file and therefore reads the bytecode from that file.
A cat would work as well, as this would directly provide the bytecode to the command. Certainly the fact that both options are valid could be somehow confusing :-)

Reply:
Oh interesting... When I used `echo`, it returns the bytecode but also the name of the file...

```bash
$ echo ./bytecode.txt | polycli wrap-contract
0x6300000007630000001560003963000000076000f3./bytecode.txt
```

Reply:
totally right!! uploaded new PR with the fix:
https://github.com/0xPolygon/polygon-cli/pull/456